src/components/accounts/CreateAccount.js

```diff
@@ -91,8 +92,10 @@ class CreateAccount extends Component {
 
     handleCheckNearDropBalance = async () => {
         try {
+            Mixpanel.track("CA Check near drop balance")
             await this.props.checkNearDropBalance(this.props.fundingContract, this.props.fundingKey)
         } catch(e) {
+            Mixpanel.track("CA Get near drop balance error", {eror: e})
```

Comment:
`eror` looks like typo. Should be `error`?

also here and in other places – I wonder what actually ends up being passed to mixpanel. Like does mixpanel know how to handle errors specifically or does it just gets serialized to JSON as regular JS object?

Reply:
thanks, it will handle error as Json